d3d/D3D12EnhancedBarriers.md

```diff
@@ -61,17 +61,17 @@ This document proposes an enhanced D3D12 Barrier API/DDI design that is capable
   - [`D3D12_TEXTURE_BARRIER`](#d3d12_texture_barrier)
   - [`D3D12_BUFFER_BARRIER`](#d3d12_buffer_barrier)
   - [`D3D12_BARRIER_GROUP`](#d3d12_barrier_group)
-  - [ID3D12GraphicsCommandList7 Barrier](#id3d12graphicscommandlist7-barrier)
-  - [ID3D12VideoDecodeCommandList3 Barrier](#id3d12videodecodecommandlist3-barrier)
-  - [ID3D12VideoProcessCommandList3 Barrier](#id3d12videoprocesscommandlist3-barrier)
-  - [ID3D12VideoEncodeCommandList3 Barrier](#id3d12videoencodecommandlist3-barrier)
-  - [ID3D12Device10 CreateCommittedResource3](#id3d12device10-createcommittedresource3)
-  - [ID3D12Device10 CreatePlacedResource2](#id3d12device10-createplacedresource2)
-  - [ID3D12Device10 CreateReservedResource2](#id3d12device10-createreservedresource2)
-  - [ID3D12DebugCommandQueue1 AssertResourceAccess](#id3d12debugcommandqueue1-assertresourceaccess)
-  - [ID3D12DebugCommandQueue1 AssertTextureLayout](#id3d12debugcommandqueue1-asserttexturelayout)
-  - [ID3D12DebugCommandList3 AssertResourceAccess](#id3d12debugcommandlist3-assertresourceaccess)
-  - [ID3D12DebugCommandList3 AssertTextureLayout](#id3d12debugcommandlist3-asserttexturelayout)
+  - [`ID3D12GraphicsCommandList7::Barrier()`](#id3d12graphicscommandlist7barrier)
+  - [`ID3D12VideoDecodeCommandList3::Barrier()`](#id3d12videodecodecommandlist3barrier)
+  - [`ID3D12VideoProcessCommandList3::Barrier()`](#id3d12videoprocesscommandlist3barrier)
+  - [`ID3D12VideoEncodeCommandList3::Barrier()`](#id3d12videoencodecommandlist3barrier)
+  - [`ID3D12Device10::CreateCommittedResource3()`](#id3d12device10createcommittedresource3)
+  - [`ID3D12Device10::CreatePlacedResource2()`](#id3d12device10createplacedresource2)
+  - [`ID3D12Device10::CreateReservedResource2()`](#id3d12device10createreservedresource2)
+  - [`ID3D12DebugCommandQueue1::AssertResourceAccess()`](#id3d12debugcommandqueue1assertresourceaccess)
+  - [`ID3D12DebugCommandQueue1::AssertTextureLayout()`](#id3d12debugcommandqueue1asserttexturelayout)
+  - [`ID3D12DebugCommandList3::AssertResourceAccess()`](#id3d12debugcommandlist3assertresourceaccess)
+  - [`ID3D12DebugCommandList3::AssertTextureLayout()`](#id3d12debugcommandlist3asserttexturelayout)
```

Comment:
Hi, can you revert the change to the header names? We mirror these specs into an internal ADO repo as well, and ADO makes... _interesting_ choices about how it converts markdown headers to anchors. Making the header look nice ends up giving an anchor like this in ADO: `%60id3d12graphicscommandlist7%3A%3Abarrier()%60`. Other specs also link to some of these, so changing the anchor's can be hazardous even when it doesn't include special characters (ex: [Advanced Texture Ops](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/HLSL_SM_6_7_Advanced_Texture_Ops.md?plain=1#L151-L153))

Reply:
Good catch, I would have had to update those as well.  Reverted now and your suggestion made me discover some similarly broken links in `HLSL_SHaderModel6_7.md` and `VulkanOn12.md`.

Note that I couldn't find any link reference to https://github.com/microsoft/DirectX-Specs/blob/master/d3d/HLSL_SM_6_7_Advanced_Texture_Ops.md, is this file supposed to be reachable from anywhere (e.g. like `HLSL_SM_6_7_Wave_Ops_Include_Helper_Lanes.md`)?

Reply:
Oh nice, thanks for finding the others too!
Looks like Advanced Texture Ops should be reachable via homepage -> Shader Model 6.7 -> Advanced Texture Ops documentation link (relative address).

Reply:
I may have made a mess out of case-sensitive search then, the links are indeed there!